GVRf/Framework/src/org/gearvrf/GVRViewManager.java

```diff
@@ -462,6 +463,11 @@ void onDrawEyeView(int eye, float fovDegrees) {
         if (!(mSensoredScene == null || !mMainScene.equals(mSensoredScene))) {
             GVRCameraRig mainCameraRig = mMainScene.getMainCameraRig();
 
+            if(eye == 0) {
+                GVRPerspectiveCamera centerCamera = mainCameraRig.getCenterCamera();
+                cull(mMainScene.getNative(), centerCamera.getNative(), mRenderBundle.getMaterialShaderManager().getNative());
+            }
+
```

Comment:
I actually did that initially, but was seeing weird things happening.  Objects would flicker in and out of existence right in front of me.  It was wierd.  Moved it to here and everything worked perfectly.  I agree this isn't the cleanest.  Let fight with it a little more and see if i can get it working from onDrawFrame.